ui/src/app/pages/admin/workspace/basic-information.tsx

```diff
@@ -1,62 +1,149 @@
+        <WorkspaceInfoField labelText='Workspace Name'>
+          {workspace.name}
+        </WorkspaceInfoField>
+        <WorkspaceInfoField labelText='Terra Name (often incorrectly called "id")'>
+          {workspace.id}
+        </WorkspaceInfoField>
+        <WorkspaceInfoField labelText='Workspace Namespace'>
+          {workspace.namespace}
+        </WorkspaceInfoField>
+        <WorkspaceInfoField labelText='Access Tier'>
+          {workspace.accessTierShortName?.toUpperCase()}
+        </WorkspaceInfoField>
+        <WorkspaceInfoField labelText='Google Project Id'>
+          {workspace.googleProject}
+        </WorkspaceInfoField>
+        <WorkspaceInfoField labelText='Billing Account Type'>
+          {isUsingFreeTierBillingAccount(workspace)
+            ? 'Initial credits'
+            : 'User provided'}
+        </WorkspaceInfoField>
+        <WorkspaceInfoField labelText='Creation Time'>
+          {new Date(workspace.creationTime).toDateString()}
+        </WorkspaceInfoField>
+        <WorkspaceInfoField labelText='Last Modified Time'>
+          {new Date(workspace.lastModifiedTime).toDateString()}
+        </WorkspaceInfoField>
+        <WorkspaceInfoField labelText='Workspace Published'>
+          <Select
+            key={featuredCategory || 'placeholder'}
+            value={featuredCategory || null}
+            placeholder='Select a category...'
+            isDisabled={featuredCategoryLoading}
+            options={[
+              {
+                value: FeaturedWorkspaceCategory.DEMO_PROJECTS,
+                label: 'Demo Projects',
+              },
+              {
+                value: FeaturedWorkspaceCategory.PHENOTYPE_LIBRARY,
+                label: 'Phenotype Library',
+              },
+              {
+                value: FeaturedWorkspaceCategory.TUTORIAL_WORKSPACES,
+                label: 'Tutorial Workspaces',
+              },
+              {
+                value: FeaturedWorkspaceCategory.COMMUNITY,
+                label: 'Community',
+              },
+            ]}
+            onChange={(v) => setFeaturedCategory(v)}
+          />
+          <Button
+            type='primary'
+            disabled={
+              featuredCategoryLoading ||
+              !featuredCategory ||
+              featuredCategory === workspace.featuredCategory
+            }
+            onClick={() => {
+              setFeaturedCategoryLoading(true);
+              workspaceAdminApi()
+                .publishWorkspaceViaDB(workspace.namespace, {
+                  category: featuredCategory,
+                })
+                .then(async () => {
+                  await reload();
+                });
+            }}
+          >
+            Publish
+          </Button>
+          <Button
+            type='secondaryOutline'
+            disabled={featuredCategoryLoading || !workspace.featuredCategory}
+            onClick={() => {
+              setFeaturedCategory(undefined);
```

Comment:
Curious Why not null?

Reply:
This was a mistake on my part. Thanks for catching it.